src/control/Control.Layers.js

```diff
@@ -113,6 +113,27 @@ L.Control.Layers = L.Control.extend({
 		return (this._map) ? this._update() : this;
 	},
 
+	// @method expand(): undefined

```

Comment:
Maybe return `this`, like most of our public methods which would otherwise be `void` do?